strip-erroneous-labels.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
```

Comment:
Even if this file isn't meant to stick around for long, can you add a line of comment here saying what this is for or reference an issue?

Reply:
Done.